compiler/passes/convert-uast.cpp

```diff
@@ -337,8 +533,17 @@ struct Converter {
   /// Calls ///
 
   Expr* visit(const uast::Dot* node) {
-    INT_FATAL("TODO");
-    return nullptr;
+    // These are the arguments that 'buildDotExpr' requires.
+    BaseAST* base = toExpr(convertAST(node->calledExpression()));
+    const char* member = node->field().c_str();
+
+    if (!strcmp(member, "type")) {
```

Comment:
These should use UniqueString comparison rather than strcmp. You can add fields to the Converter that store some UniqueStrings that it uses so that they are cached. At some point we might want to have an idea of "well known unique strings" but for now let's just add some fields to cache. (Doing it here has the advantage that the strings cached by this type stay in the implementation here, rather than having some global table of all the interesting strings which both gets overwhelming and also is something we have to constantly update. But we might get tired of using the fields approach).